# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/player/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/player/Cargo.toml b/gfx/wgpu/player/Cargo.toml
deleted file mode 100644
--- a/gfx/wgpu/player/Cargo.toml
+++ /dev/null
@@ -1,36 +0,0 @@
-[package]
-name = "player"
-version = "0.1.0"
-authors = [
-	"Dzmitry Malyshau <kvark@mozilla.com>",
-]
-edition = "2018"
-description = "WebGPU trace player"
-homepage = "https://github.com/gfx-rs/wgpu"
-repository = "https://github.com/gfx-rs/wgpu"
-keywords = ["graphics"]
-license = "MIT OR Apache-2.0"
-publish = false
-
-[features]
-cross = ["wgc/cross"]
-
-[dependencies]
-env_logger = "0.8"
-log = "0.4"
-raw-window-handle = "0.3"
-ron = "0.6"
-winit = { version = "0.24", optional = true }
-
-[dependencies.wgt]
-path = "../wgpu-types"
-package = "wgpu-types"
-features = ["replay"]
-
-[dependencies.wgc]
-path = "../wgpu-core"
-package = "wgpu-core"
-features = ["replay", "raw-window-handle"]
-
-[dev-dependencies]
-serde = "1"